docs/using.md

```diff
@@ -163,10 +200,14 @@ In this example, there is a single `HelloWorld` actor allocated two threads. Thi
       Type: InsertRemove
       Threads: 100
       Phases:
-      - Duration: 10 milliseconds
-      - Nop
+      - Collection: inserts
+		Database: test
+	    Duration: 10 milliseconds
+      - Nop: true
+
+This example has an additional `InsertRemove` actor with 100 threads, where each thread inserts and removes a document as fast as possible. A user observing the database contents might notice a document appearing and disappearing rapidly. This actor will output `InsertRemoveExample.Insert.ftdc` and `InsertRemoveExample.Remove.ftdc` time series outputs, showing the insertions and removals happening for 10 milliseconds at the phase start. (See [here](#orgec88ad4) for more details on outputs.)
 
-This example has an additional `InsertRemove` actor with 100 threads, where each thread inserts and removes a document as fast as possible. Note that even though the actors are listed sequentially, all actors are concurrent.
+Note that even though the actors are listed sequentially, all actors are concurrent.
```

Comment:
Could add a note saying that using nop phases is the common pattern for modeling sequential phases (i.e. loading first, then doing crud). Up to you if it's worth though, and you may already mention something similar later on

Reply:
I'd also link heavily to example workloads. In particular the [multiple phases example](https://github.com/mongodb/genny/blob/master/src/workloads/docs/HelloWorld-MultiplePhases.yml) I think is relevant here.